test/itemrenderer.cpp

```diff
@@ -713,4 +713,24 @@ TEST_CASE("item_renderer::render_plaintext() splits text on newlines", "[item_re
 			"aliqua."
 		});
 	}
+
+	SECTION("render_plaintext escapes angle brackets for STFL") {
+		const std::string text = "this < should be escaped >\n<p>test text</p>";
+
+		check(text, {
+			"this <> should be escaped >",
+			"<>p>test text<>/p>"
+		});
+	}
+
+	SECTION("render_plaintext does not escape when rendering towards some plaintext output") {
+		const std::string text = "this < should *not* be escaped >\n<p>test text</p>";
+
+		const bool render_for_stfl = false;
```

Comment:
By the way, have you heard of "boolean blindness"? Newsboat has much, much more important issues, so I rarely think about this stuff, but it'd probably be nicer to use enums in place of bools, even when there are only two options. It'd be more readable even without creating local constants like this one (which I believe we don't always bother to do).

Reply:
Never heard of it before.

I'm now using an enum, is this what you had in mind? (https://github.com/newsboat/newsboat/commit/799baac5d8390fff5b129c138b1c80b1d562ee60)

Reply:
Yes, exactly! I didn't mean it as a request for change, just wanted to know what you think of this approach. Should've labeled it "discussion" or something. So: do you think this approach makes more sense than `bool`s? (I do, but I'm always enthusiastic about type safety stuff even when it makes very little difference, so I'd like a second opinion.)

Reply:
I don't really have an opinion on this but I guess it does help readability in some cases (including this place).